applications/GeoMechanicsApplication/tests/test_partially_saturated/test_saturated_below_phreatic_level_pw_triangle3N/column.mdpa

```diff
@@ -0,0 +1,206 @@
+   11   0.0000000000  -2.5000000000   0.0000000000
+   12   1.0000000000  -2.5000000000   0.0000000000
+   13   0.0000000000  -2.0000000000   0.0000000000
+   14   1.0000000000  -2.0000000000   0.0000000000
+   15   0.0000000000  -1.5000000000   0.0000000000
+   16   1.0000000000  -1.5000000000   0.0000000000
+   17   0.0000000000  -1.0000000000   0.0000000000
+   18   1.0000000000  -1.0000000000   0.0000000000
+   19   0.0000000000  -0.5000000000   0.0000000000
+   20   1.0000000000  -0.5000000000   0.0000000000
+   21   0.0000000000   0.0000000000   0.0000000000
+   22   1.0000000000   0.0000000000   0.0000000000
+End Nodes
+
+
+Begin Elements SteadyStatePwElement2D3N // GUI group identifier: Grond
+        1          0     1     2     4
+        2          0     4     3     1
+        3          0     3     4     6
+        4          0     6     5     3
+        5          0     5     6     8
+        6          0     8     7     5
+        7          0     7     8    10
+        8          0    10     9     7
+        9          0     9    10    12
+       10          0    12    11     9
+       11          0    11    12    14
+       12          0    14    13    11
+       13          0    13    14    16
+       14          0    16    15    13
+       15          0    15    16    18
+       16          0    18    17    15
+       17          0    17    18    20
+       18          0    20    19    17
+       19          0    19    20    22
+       20          0    22    21    19
+End Elements
+
+Begin SubModelPart Parts_Solid_Grond // Group Grond // Subtree Parts_Solid
+    Begin SubModelPartNodes
+            1
+            2
+            3
+            4
+            5
+            6
+            7
+            8
+            9
+           10
+           11
+           12
+           13
+           14
+           15
+           16
+           17
+           18
+           19
+           20
+           21
+           22
+    End SubModelPartNodes
+    Begin SubModelPartElements
+            1
+            2
+            3
+            4
+            5
+            6
+            7
+            8
+            9
+           10
+           11
+           12
+           13
+           14
+           15
+           16
+           17
+           18
+           19
+           20
+    End SubModelPartElements
+End SubModelPart
+
+Begin SubModelPart gravity
+    Begin SubModelPartNodes
+            1
+            2
+            3
+            4
+            5
+            6
+            7
+            8
+            9
+           10
+           11
+           12
+           13
+           14
+           15
+           16
+           17
+           18
+           19
+           20
+           21
+           22
+    End SubModelPartNodes
+End SubModelPart
+
+Begin SubModelPart DISPLACEMENT_Bottom // Group Bottom // Subtree DISPLACEMENT
+    Begin SubModelPartNodes
+            1
+            2
+    End SubModelPartNodes
+End SubModelPart
```

Comment:
This sub-model part is not being used in this simulation, so I would take it out.

Reply:
Done